torch/onnx/experimental/fonnx/v1/export_options.py

```diff
@@ -0,0 +1,13 @@
+@dataclasses.dataclass
+class ExportOptions:
+    """Class for keeping track of ONNX exporter flags can perform."""
+
+    opset_version: Optional[int] = 17
+    """opset_version: Target ONNX opset version"""
+
+    dynamic_shape: Optional[bool] = True
```

Comment:
```suggestion
    dynamic_shape: bool = True
```

Reply:
Interesting, when a default value is present, isn't recommended to have the `Optional` hint? If not, we may need to change `opset_version: Optional[int] = 17` too